packages/pluggableWidgets/radio-buttons-native/src/RadioButtons.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.native.radiobuttons.RadioButtons"
+        pluginWidget="true"
+        needsEntityContext="true"
+        offlineCapable="true"
+        supportedPlatform="Native"
+        xmlns="http://www.mendix.com/widget/1.0/"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../node_modules/mendix/custom_widget.xsd">
+    <name>Radio Buttons</name>
+    <description/>
+    <studioProCategory>Input elements</studioProCategory>
+    <studioCategory>Input elements</studioCategory>
+    <icon/>
+    <properties>
+    <propertyGroup caption='General'>
+        <propertyGroup caption="Data Source">
+            <property key="enum" type="attribute" >
```

Comment:
It looks like this file isn't formatted correctly :)

Reply:
Done